fixtures/import-npm/package.json

```diff
@@ -7,9 +7,10 @@
 		"packages/*"
 	],
 	"scripts": {
-		"check:type": "rm -rf node_modules && npm install && npm run check:type --workspaces",
-		"test:ci": "npm install && npm run test:ci --workspaces",
-		"test:watch": "npm install && npm run test:watch --workspaces",
-		"type:tests": "rm -rf node_modules && npm install && npm run type:tests --workspaces"
+		"_clean_install": "rm -rf node_modules && npm install",
+		"check:type": "npm run _clean_install && npm run check:type --workspaces",
+		"test:ci": "npm run _clean_install && npm run test:ci --workspaces",
+		"test:watch": "npm run _clean_install && npm run test:watch --workspaces",
+		"type:tests": "npm run _clean_install && npm run type:tests --workspaces"
```

Comment:
I found that this fixture's tests kept failing; and I think it was because we need to ensure that the node_modules directory was in fact created by the `npm i` command here, and not a `pnpm i` command that is naturally run when setting up the workers-sdk project.

Reply:
Furthemore I couldn't just run _clean_install in front of each of these scripts because turbo tries to run then in parallel. So the solution was to add a turbo dependency instead, which means that this only runs once.